DEVELOPMENT.md

```diff
@@ -0,0 +1,60 @@
+# Development
+
+Thank you for your interest in developing on TypeScript ESLint! ❤️‍🔥
+
+> See [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our general contribution flows.
+
+## Setup
+
+After [forking the repo from GitHub](https://help.github.com/articles/fork-a-repo):
+
+```shell
+git clone https://github.com/<your-name-here>/typescript-eslint
+cd typescript-eslint
+yarn
+yarn build
+```
+
+## Validating Changes
```

Comment:
Probably worth adding a section about building?
- explicitly mention `yarn build`
- mention that you need to build for changes to propagate to other packages (i.e. make a change in scope-manager, then build, then test in eslint-plugin)